data/helpers.d/php

```diff
@@ -560,3 +560,63 @@ ynh_get_scalable_phpfpm () {
         fi
     fi
 }
+
+readonly YNH_DEFAULT_COMPOSER_VERSION=1.10.17
```

Comment:
What's the reasons not to have composer 2.0.8 as default version (the current stable version) ?
Probably a naive question i know :)

Reply:
When we updated the helpers there were multiple common Composer packages that were yet to be compatible with v2. That was breaking our apps... so we sticked to v1.

I will try to check tomorrow if they are updated. There was one from Wikimedia for example.

Reply:
Because mostly app still use composer 1 ^^